src/main/java/com/rakovets/course/java/core/practice/generics/math/Math.java

```diff
@@ -0,0 +1,92 @@
+package com.rakovets.course.java.core.practice.generics.math;
+
+import java.util.Arrays;
+
+public abstract class Math {
+    private static Number[] arrayMath;
+
+    public static  <T extends Number> Number minOfFive(T e1, T e2, T e3, T e4, T e5) {
```

Comment:
```suggestion
    public static  <T extends Number> Number min(T e1, T e2, T e3, T e4, T e5) {
```

USE overloading, DON'T specific method name